### DISCUSSION TITLE
Multiple repositories for the same sources

### DISCUSSION THREAD

**Discussion by fdw** [rustic-rs/rustic#655] [Ideas]
I'm backup up the same (or at least similar) data to a local HD and to a cloud server. If I read the config files right, this would mean I need one for the cloud and one for local, even though they share some parameters. Do you think it would be possible to have some way to backup the same sources to different repositories with less duplication?

Bonus if the forget rules could change based on the repo, too - locally, I keep backups for much longer than in the cloud.

**Comment by aawsome** [comment]
If you are talking about config files, yes, for different setups you need different config files. But you can put common options in another config file, e.g. `common.toml` and source them using `use-profile = ["common"]`. Note that you can source from multiple other profiles and even do so recursively, i.e. you could also use a `use-profile` within `common.toml`.

Another remark which might help in your setup: Have a look at the `copy` command which can be used to copy snapshots to another repository.

**Comment by aawsome** [comment]
Oh, just forgot: Of course, you can also have multiple config files and use them all with the CLI, e.g. `rustic -P common -P local <command>`
